api/envoy/config/core/v3/extension.proto

```diff
@@ -28,3 +30,30 @@ message TypedExtensionConfig {
   // <config_overview_extension_configuration>` for further details.
   google.protobuf.Any typed_config = 2 [(validate.rules).any = {required: true}];
 }
+
+// Configuration source specifier for a late-bound
+// extension configuration. The parent resource is warmed until all the initial
+// extension configurations are received, unless the flag to apply the default
+// configuration is set. Subsequent extension updates are atomic on a per-worker
+// basis, and apply to new streams while the active streams continue using
+// the older extension configurations. If the initial delivery of the extension
+// configuration fails, due to a timeout for example, the optional default
+// configuration is applied. Without a default configuration, the extension is
+// disabled, until an extension configuration is received.
```

Comment:
What does it mean for an extension to be disabled? I suppose this is context specific? Maybe clarify that somehow?